config/locales/en.yml

```diff
@@ -268,6 +268,8 @@ en:
           edit: Trainee personal details
         trainee_start_date:
           edit: &commencement_date Trainee’s start date
+        trainee_start_status:
+          edit: &did_trainee_start_itt_on_time Did the trainee start their ITT on time?
```

Comment:
ITT needs a `<abbr>` tag

Reply:
This one is the page title, won't be able to use html in here.
![image](https://user-images.githubusercontent.com/910019/143864722-4095f758-1b11-4f43-b836-7e400290e441.png)